pkg/network/setup/hotplugcontroller_test.go

```diff
@@ -0,0 +1,49 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/tests/libvmi"
```

Comment:
I do not think it is allowed to import from the e2e tests.

Reply:
I can only comment on things I manage to review, the fact the such dependencies have been introduced is unfortunate.

> Why would you say that ? Was this ever discussed ?

I think the burden should be on the one attempting to import e2e test libraries from unit tests.
I do not think it is reasonable for a unit test to depend on e2e test, which are to exposed formally as an API.
Anyone can change the libraries there and the expectations is for it to affect the e2e tests artifact only and not something else.

The e2e tests are to be kept isolated and self contained. No one should depend on them or their libs.
I only now realized that this is not that trivial.

Reply:
AFAICT I am no longer introducing this dependency; thus resolving this issue.